From 71c3b3f779c815d5b1c0a87e6d09dadefd6b7409 Mon Sep 17 00:00:00 2001
From: Wolfgang Bauer <wbauer@tmo.at>
Date: Tue, 11 Jul 2017 14:25:19 +0200
Subject: Support SVG too

Konqueror's about page doesn't show icons if the breeze icon theme is
used.
The reason is that khtml doesn't support SVG images.
This patch is supposed to fix it by allowing to use Qt's support in that
case.

This is a backport of commit ed5c039f2d1996545b95a24e9bff4264144f9f6d
in khtml

FIXED-IN: 4.14.35
CCBUG: 355872
---
 khtml/imload/decoders/qimageioloader.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/khtml/imload/decoders/qimageioloader.cpp b/khtml/imload/decoders/qimageioloader.cpp
index cecaeaf..7b6025b 100644
--- a/khtml/imload/decoders/qimageioloader.cpp
+++ b/khtml/imload/decoders/qimageioloader.cpp
@@ -131,7 +131,7 @@ public:
 };
 
 static const char* const positiveList[] = {
-    "BMP", "TIFF", "JP2", "PNM", "EXR", "XBM", "XPM", "ICO", 0
+    "BMP", "TIFF", "JP2", "PNM", "EXR", "XBM", "XPM", "ICO", "SVG", "SVGZ", 0
 };
 
 bool isSupportedFormat(QString format) {
-- 
cgit v0.11.2
